app/views/admin/bikes/edit.html.haml

```diff
@@ -20,7 +20,7 @@
         - if @bike.current_stolen_record.present?
           = @bike.dl_list_item(@bike.current_stolen_record.phone, 'Phone')
       %dl.dl-horizontal
-        = @bike.dl_list_item(@bike.created_at.strftime('%m-%d-%Y'), 'Created')
+        = @bike.dl_list_item(@bike.created_at.strftime('%Y-%m-%d'), 'Created')
```

Comment:
We don't need to change the admin sections. While it might be useful, there is some js ordering stuff that uses the date - the dates are in the hidden spans - and changing that would have negative impacts.

Plus, it's the admin section, it isn't user facing.